inginious/frontend/templates/course_admin/task_list.html

```diff
@@ -113,4 +116,16 @@ <h5 class="modal-title" id="taskDispenserModalLabel">{{_("Switch task dispenser"
         </button>
     </div>
 </div>
+
+<script>
+    const tasksWithErrors = document.getElementsByClassName("bg-danger")
+
+    for(let task of tasksWithErrors){
```

Comment:
This still makes no sense to do this with javascript code. This can be done in the Jinja template using bootstrap alerts and relocated on top of the page.